### ISSUE TITLE
Training scripts for pre-trained models?

### ISSUE THREAD

**Issue by antspy** [pytorch/vision#286]
Hello, 

Is there a way to access the training script with which the resnet models were trained?

Thank you!

**Comment by fmassa** [comment]
Yes, they were all trained using the same script https://github.com/pytorch/examples/tree/master/imagenet

**Comment by D-X-Y** [comment]
@fmassa  Would you mind providing some detailed hyper-parameters such as number of GPUs, LR, weight decay? Because the parameters for MobileNet and ShuffleNet are quite different from the family of ResNets.

**Comment by fmassa** [comment]
Yes, they will all be present in https://github.com/pytorch/vision/tree/master/references/classification

**Comment by D-X-Y** [comment]
Thanks. Look forward to the future references

**Comment by RuiWang1998** [comment]
Did you guys use seed

> Yes, they were all trained using the same script https://github.com/pytorch/examples/tree/master/imagenet

Did you guys use —seed?

**Comment by fmassa** [comment]
@RuiWang1998 for newer models, they have implemented using the training scripts from https://github.com/pytorch/vision/tree/master/references , and we do not fix the random seed by default.
